content/questions/actions/question-076.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "How many reusable workflows can you call from a single workflow file? "
```

Comment:
Let's rephrase this to say "maximum" because right now all answers are correct

```suggestion
title: "What's the maximum amount of reusable workflows can you call from a single workflow file? "
```